website/docs/r/security_center_workspace.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_security_center_workspace"
+sidebar_current: "docs-azurerm-security-center-workspace"
+description: |-
+    Manages the subscription's Security Center Workspace.
+---
+
+# azurerm_security_center_workspace
+
+Manages the subscription's Security Center Workspace.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "tfex-security-workspace"
+  location = "westus"
+}
+
+resource "azurerm_log_analytics_workspace" "example" {
+  name                = "tfex-security-workspace"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  sku                 = "PerGB2018"
+}
+
+resource "azurerm_security_center_contact" "example" {
+  scope        = "/subscriptions/00000000-0000-0000-0000-000000000000"
+  workspace_id = "${azurerm_log_analytics_workspace.example.id}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `scope` - (Required) The scope of VMs to send their security data to the desired workspace, unless overridden by a setting with more specific scope.
```

Comment:
is this just VM's, or is this Resources?

Reply:
The API docs specifically said VMs